front/app/containers/Admin/invitations/routes.tsx

```diff
@@ -2,12 +2,24 @@ import React, { lazy } from 'react';
 
 import PageLoading from 'components/UI/PageLoading';
 
+import { AdminRoute } from '../routes';
+import { UsersRoute } from '../users/routes';
+
 const AdminInvitationsContainer = lazy(() => import('.'));
 const AdminInvitationsInvite = lazy(() => import('./invite'));
 const AdminInvitationsAll = lazy(() => import('./all'));
 
+enum invitationRoutes {
+  invitations = 'users/invitations',
+  all = 'all',
+}
+
+export type invitationRouteTypes =
+  | AdminRoute<invitationRoutes.invitations>
+  | UsersRoute<`${invitationRoutes.all}`>;
+
```

Comment:
@EdwinKato Should line 19 be `admin/users/invitations/all`? I think right now it's just `users/all` ? :thinking: